packages/syft/src/syft/service/user/user_service.py

```diff
@@ -317,7 +319,9 @@ def update(
 
         return user.to(UserView)
 
-    def get_target_object(self, credentials: SyftVerifyKey, uid: UID):
+    def get_target_object(
+        self, credentials: SyftVerifyKey, uid: UID
+    ) -> Union[SyftError, User]:
```

Comment:
nitpick: usually the convention is that the error type is on the right and the success type is on the left.

```suggestion
    ) -> Union[User, SyftError]:
```